codec/decode.go

```diff
@@ -45,6 +46,7 @@ func Decode(in []byte, t interface{}) (interface{}, error) {
 	return output, err
 }
 
+// DecodePtr pointer
```

Comment:
DecodePtr decodes the input into a pointer to an interface ?

Reply:
will update this and try to provide better description to others.